doc/integrator/docker.rst

```diff
@@ -131,3 +131,25 @@ something like that:
 
     environment:
       DB_CONNECTION: user=www-data password=toto dbname=geoacordaDev host=db
+
+
+Keep your DB schema up to date
+------------------------------
+
+The WSGI image contains Alembic. You can use it as a start once container and
+add something like that in your composition:
+
+.. code:: yaml
+
+    alembic:
+      labels:
+        io.rancher.container.start_once: 'true'
```

Comment:
Yes, it's what rancher is generating when you use their GUI to edit that. So they must expect a string, I guess.